.github/workflows/deploy-gke-clusters.yaml

```diff
@@ -70,23 +70,23 @@ jobs:
     #   run: ./scripts/deploy-cluster-with-charon-tag.sh charon-prater-0 ${{ steps.charon_image_tag.outputs.tag }}
     #   shell: bash
 
-    - name: Delete charon-prater-1
-      run: ./scripts/delete-cluster.sh charon-prater-1
-      shell: bash
-    - name: Deploy charon-prater-1
-      run: ./scripts/deploy-cluster-with-charon-tag.sh charon-prater-1 ${{ steps.charon_image_tag.outputs.tag }}
-      shell: bash
+    # - name: Delete charon-prater-1
+    #   run: ./scripts/delete-cluster.sh charon-prater-1
+    #   shell: bash
+    # - name: Deploy charon-prater-1
+    #   run: ./scripts/deploy-cluster-with-charon-tag.sh charon-prater-1 ${{ steps.charon_image_tag.outputs.tag }}
+    #   shell: bash
 
-    - name: Delete charon-sepolia-1780
-      run: ./scripts/delete-cluster.sh charon-sepolia-1780
-      shell: bash
-    - name: Deploy charon-sepolia-1780
-      run: ./scripts/deploy-cluster-with-charon-tag.sh charon-sepolia-1780 ${{ steps.charon_image_tag.outputs.tag }}
-      shell: bash
+    # - name: Delete charon-sepolia-1780
+    #   run: ./scripts/delete-cluster.sh charon-sepolia-1780
+    #   shell: bash
+    # - name: Deploy charon-sepolia-1780
+    #   run: ./scripts/deploy-cluster-with-charon-tag.sh charon-sepolia-1780 ${{ steps.charon_image_tag.outputs.tag }}
+    #   shell: bash
 
-    - name: Delete canary-holesky-1
-      run: ./scripts/delete-cluster.sh canary-holesky-1
-      shell: bash
-    - name: Deploy canary-holesky-1
-      run: ./scripts/deploy-cluster-with-charon-tag.sh canary-holesky-1 ${{ steps.charon_image_tag.outputs.tag }}
-      shell: bash
+    # - name: Delete canary-holesky-1
+    #   run: ./scripts/delete-cluster.sh canary-holesky-1
+    #   shell: bash
+    # - name: Deploy canary-holesky-1
+    #   run: ./scripts/deploy-cluster-with-charon-tag.sh canary-holesky-1 ${{ steps.charon_image_tag.outputs.tag }}
+    #   shell: bash
```

Comment:
is canary holesky 1 idle? It seems like it's just perpetually broken...?

Reply:
yes because we moved all test clusters from gke to aws